mirage/scenarios/default.ts

```diff
@@ -16,7 +16,16 @@ const {
 export default function(server: Server) {
     const currentUser = server.create('user', 'loggedIn');
     const firstNode = server.create('node', {});
-    server.create('contributor', { node: firstNode, users: currentUser, index: 0 });
+    server.create('contributor', { node: firstNode, users: currentUser, bibliographic: true });
+
+    const nodeManyContribs = server.create('node', { id: 'x2pqs' });
+    server.create('contributor', { node: nodeManyContribs, users: currentUser, bibliographic: true });
+    server.create('contributor', { node: nodeManyContribs, bibliographic: true });
```

Comment:
You could compress this a bit by doing something like:
```ts
server.createList('contributor', 2, { node: nodeManyContribs, bibliographic: true });
server.createList('contributor', 3, { node: nodeManyContribs, bibliographic: false });
```
Could I also request that this node have more than 10 contributors? Right now the `showMore` example in the handbook doesn't do anything because there's no more to show.